Helper/Data.php

```diff
@@ -298,9 +298,9 @@ public function consignmentHasShipmentOption(AbstractConsignment $consignment, s
      */
     public function convertToGrams(?float $weight): int
     {
-        $weightType = $this->getGeneralConfig('print/weight_indication');
+        $weightType = $this->getGeneralConfig('general/locale/weight_unit');
```

Comment:
Traditionally we have used our own setting, please find out why that wouldn’t work and this ‘breaking’ change is necessary. Additionally, the own setting is also used in `PackageRepository.php`, I think that should be involved in this PR as well then.